docs/datadog_dashboard.md

```diff
@@ -0,0 +1,135 @@
+# Datadog Dashboards
+This feature is in preview.
+
+## Overview
+The DatadogDashboard Custom Resource Definition (CRD) allows users to create [dashboards][1] using the Operator and manage them as Kubernetes resources.
+
+## Prerequisites
+
+- Datadog Operator v1.9+
+- **[Helm][2]** for deploying the Datadog Operator
+- **[`kubectl` CLI][3]** for installing a `DatadogDashboard`
+
+Tests were performed on Kubernetes versions >= `v1.30.0`
+
+## Adding a DatadogDashboard
+
+To deploy a `DatadogDashboard` with the Datadog Operator, use the [`datadog-operator` Helm chart][3].
```

Comment:
```suggestion
To deploy a `DatadogDashboard` with the Datadog Operator, use the [`datadog-operator` Helm chart][4].
```

Reply:
Not sure if you looked but rendering hyperlink with code and text mix looks kind of broken 
  
<img width="211" alt="Screenshot 2024-09-30 at 11 28 50 AM" src="https://github.com/user-attachments/assets/8c6d85f6-c3b0-4155-ae0b-0f8e32264647">